src/test/java/core/basesyntax/handlers/BalanceHandlerTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.storage.Storage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceHandlerTest {
+    private static final String FRUIT_NAME = "banana";
+    private static final int CORRECT_QUANTITY = 50;
+    private static final int ZERO_QUANTITY = 0;
+    private static final int NEGATIVE_QUANTITY = -10;
+    private static final String EXCEPTION_MESSAGE_NEGATIVE_QUANTITY =
+            "Quantity of fruit can`t be a negative";
+
+    private OperationHandler balanceHandler;
+
+    @BeforeEach
+    void setUp() {
+        balanceHandler = new BalanceHandler();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    void handle_correctQuantity_ok() {
+        balanceHandler.handle(FRUIT_NAME, CORRECT_QUANTITY);
```

Comment:
Please, add empty lines between logical parts of the test method (setup logic, executions, assertions)